ExtendedRelationOntology.ttl

```diff
@@ -42,6 +44,32 @@ cco:code_license rdf:type owl:AnnotationProperty ;
                  cco:is_curated_in_ontology "http://www.ontologyrepository.com/CommonCoreOntologies/Mid/ExtendedRelationOntology"^^xsd:anyURI ;
                  rdfs:label "code license"@en .
 
+###  http://www.ontologyrepository.com/CommonCoreOntologies/is_disrupted_by
+cco:is_disrupted_by rdf:type owl:ObjectProperty ;
+                    dcterms:created "2022-12-30T21:32:27-05:00"^^xsd:dateTime ;
+                    owl:inverseOf cco:is_disrupted_by ;
+                    rdfs:domain obo:BFO_0000015 ;
+                    rdfs:range obo:BFO_0000015 ;
+                    cco:definition "Inverse of disrupts." ;
+                    dcterms:creator "https://cubrc.org"^^xsd:anyURI ;
+                    cco:is_curated_in_ontology "http://www.ontologyrepository.com/CommonCoreOntologies/Mid/ExtendedRelationOntology"^^xsd:anyURI ;
+                    rdfs:label "is disrupted by"@en ;
+                    skos:prefLabel "is disrupted by"@en .
+
+###  http://www.ontologyrepository.com/CommonCoreOntologies/disrupts
+cco:disrupts rdf:type owl:ObjectProperty ;
+             owl:inverseOf cco:is_disrupted_by ;
+             rdfs:domain obo:BFO_0000015 ;
+             rdfs:range obo:BFO_0000015 ;
+             dcterms:created "2022-12-30T21:32:27-05:00"^^xsd:dateTime ;
+             dcterms:creator "https://cubrc.org"^^xsd:anyURI ;
+             cco:definition "A relation where one process disrupts another process from occurring as it would have."@en ;
+             cco:elucidation "A process can disrupt another process from occurring as it would have by 1) preventing a disposition or role from realizing in that process, 2) lowering the grade of the process, or 3) stopping the process from continuing to occur."@en ;
```

Comment:
I believe 1 should read "from realizing that process" rather than "from realizing in that process"

Reply:
A process realizes a realizable entity, and process are not realizable, so isn't "preventing a role or disposition from realizing in that process" correct?

Reply:
@DavidGLimbaugh Ha I almost wrote the same thing about "realizes in that process"; both "realizes in that process" and "realizes that process" sound odd to me.  I'll update my suggestion by appealing to the alternative label for BFO_0000054, i.e. "is realized by". That is, I suggest replacing "from realizing in that process" with "from being realized by that process".

Reply:
I think that is more clear. It's been changed.